releases/tickets-source/v18_1_0.rst

```diff
@@ -0,0 +1,6 @@
+.. _release-v18-1-0-tickets:
+
+Tickets Addressed in RFC-618 Release (v18_1_0)
+==================================================
+
+- `DM-20664 <https://jira.lsstcorp.org/browse/DM-20664>`_: Backport issue DM-20506 to 18.1.x branch
```

Comment:
This description is accurate, but not very helpful. Anybody who wants to know why we made this release will have to click through multiple layers of hierarchy to get here, and then will just end up wondering what DM-20506 was all about!